example/build.gradle

```diff
@@ -39,13 +39,15 @@ android {
             minifyEnabled true
             proguardFiles getDefaultProguardFile('proguard-android.txt')
 
-            signingConfig signingConfigs.release
+            if (file(releaseKeyStoreFileName).exists() && file(releasePropFileName).exists()) {
+                signingConfig signingConfigs.release
```

Comment:
Good point - An unsigned apk is not very useful (unless you [manually sign it later](https://developer.android.com/studio/publish/app-signing.html#signing-manually)). However, issuing a simple `./gradlew build` on a fresh clone of this repo would not succeed without the keystore and properties files. I understand this is the expected behavior of the fail-fast setup, but would you be okay with something along these lines instead:

    if (file(releaseKeyStoreFileName).exists() && file(releasePropFileName).exists()) {
        signingConfig signingConfigs.release
    } else {
        println '#####################################################################'
        println '# WARNING: Incomplete signing config. Release APK will be unsigned. #'
        println '#####################################################################'
    }

It would not fail the build, but make it clear that something is missing. `./gradlew build` is one of the first commands I run on every newly cloned Android project. According to the Gradle documentation it "`Assembles and tests this project.`". So if it fails, and additional manual setup is required just to make it pass, it feels like something is wrong. :)
Let me know if you want me to add the warning or revert the commit.